portal-api/api-tests/v1/bonds/bond-change-cover-start-date.api-test.js

```diff
@@ -116,10 +119,10 @@ describe('/v1/deals/:id/bond/change-cover-start-date', () => {
       expect(status).toEqual(404);
     });
 
-    it('accepts requests if <user>.bank.id == *', async () => {
+    it('400 requests if <user>.bank.id == *', async () => {
```

Comment:
should this be 400?
if so, it should read
`it('returns 400 status if...')`

Reply:
should be 400.  Discussed with Abhi.  Not sure why the test is there, but with added validation, there are errors so should be 400

Reply:
done